src/Servers/Kestrel/test/Libuv.BindTests/Libuv.BindTests.csproj

```diff
@@ -5,7 +5,7 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <TestGroupName>Libuv.BindTests</TestGroupName>
     <!-- https://github.com/dotnet/aspnetcore/issues/22114 -->
-    <SkipHelixQueues>Windows.10.Arm64;Windows.10.Arm64.Open</SkipHelixQueues>
+    <SkipHelixQueues>Windows.10.Arm64;Windows.10.Arm64.Open;Windows.10.Arm64v8;Windows.10.Arm64v8.Open</SkipHelixQueues>
```

Comment:
Change all these to `<SkipHelixArm>true</SkipHelixArm>`?

Reply:
Do we not want (or are they not supported) on the linux arm queues?

Reply:
If we aren't using the old queue anymore, shouldn't we get rid of the Windows.10.Arm64;Windows.10.Arm64.Open entries?

Reply:
> These tests will never work on ARM

Huh❔ They seemed to run fine&hellip;
![image](https://user-images.githubusercontent.com/6431421/97741150-8b0ded00-1a9f-11eb-8cc7-98bed13b9347.png)

Reply:
Oops
![image](https://user-images.githubusercontent.com/6431421/97741277-abd64280-1a9f-11eb-9a52-fe401e3f3e51.png)